.github/workflows/codeql-workflow.yml

```diff
@@ -0,0 +1,41 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+  pull_request:
+    branches:
+      - master
+      - develop
+    paths-ignore:
+      - '.github/workflows/*.yml'
+
+jobs:
+  analyse:
+    name: Analyse
+    strategy: 
+      matrix:
+        sdk: [ "$NANOS_SDK", "$NANOX_SDK", "$NANOSP_SDK" ]
+        language: [ 'cpp' ]
```

Comment:
Maybe add a comment the language 'cpp' also covers C, this might confuse external devs

Reply:
Add in last version

Reply:
"'cpp' covers C and C++"